runtime.md

```diff
@@ -1,5 +1,51 @@
 # Runtime and Lifecycle
 
+## State
+
+The runtime state for a container is persisted on disk so that external tools can consume and act on this information.
+The runtime state is stored in a JSON encoded file.
+It is recommended that this file is stored in a temporary filesystem so that it can be removed on a system reboot.
+On Linux based systems the state information is stored in `/run/oci`.
```

Comment:
On Wed, Aug 26, 2015 at 10:37:33AM -0700, Brandon Philips wrote:
> > @@ -1,5 +1,51 @@
> >  # Runtime and Lifecycle
> >  
> > +## State
> > +
> > +The runtime state for a container is persisted on disk so that external tools can consume and act on this information.
> > +The runtime state is stored in a JSON encoded file.
> > +It is recommended that this file is stored in a temporary filesystem so that it can be removed on a system reboot.
> > +On Linux based systems the state information is stored in `/run/oci`.
> 
> @vbatts suggested that we use `/run/oci/containers/` so we don't consume the whole namespace.

In further meeting discussion, I think we stepped back from that and
collected around:

* /run/oci/{container-id} is a flat namespace for all containers,
  although the ID can contain forward slashes so it might involve
  several directories.  But there's no OCI-specified namespacing.
* Any important information (e.g. “this state is for a container, not
  a hypervisor”) will be stored in the state file itself.

I think my namespaced-version proposal [1] will make it easy to
collect information like “hypervisor vs. container”, with version
entries like:

  "version": {
    "ocsContainerState": "1.0.0"
  }

or:

  "version": {
    "runVState": "0.3.0"
  }

etc.

[1]: https://github.com/opencontainers/specs/issues/110#issuecomment-133587646